app/src/test/java/org/grammatek/simacorrect/spellcheckerservice/YfirlesturAnnotationTest.kt

```diff
@@ -0,0 +1,219 @@
+package org.grammatek.simacorrect.spellcheckerservice
+
+import com.google.common.truth.Truth.assertThat
+import org.grammatek.apis.DevelopersApi
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+typealias AnnotationIndices = YfirlesturAnnotation.AnnotationIndices
+
```

Comment:
In general, the tests would dramatically be more valuable, if we could have all raw texts in one or multiple separate file(s) and the test cases would just import this/these file(s). Then it would be easy to add new examples to the tests without touching any test source code lines.

We have e.g. error corpora that we could use to get well known error suggestions.

Reply:
That was my first approach, I did however struggle to implement it since majority of these tests have a specific value to test for as oppose to a general one. 
The question is, is it worth while to refactor the to accomplish this?

Reply:
Absolutely ! Then you can add e.g. snippets of the official annotated error corpora that exist for Icelandic and see how the underlying spell-checker performs. Also the text samples are really quite "constructed" and it's so much easier to amend real-world examples this way.